pkg/ddc/alluxio/transform.go

```diff
@@ -316,12 +319,12 @@ func (e *AlluxioEngine) transformWorkers(runtime *datav1alpha1.AlluxioRuntime, v
 	e.optimizeDefaultForWorker(runtime, value)
 
 	// labelName := common.LabelAnnotationStorageCapacityPrefix + e.runtimeType + "-" + e.name
-	labelName := e.getCommonLabelname()
+	//labelName := e.getCommonLabelname()
 
```

Comment:
Can we delete this logic code if it is no longer used?

Reply:
thanks, i will delete this no longer used logic code😁